server/games/game.py

```diff
@@ -758,6 +758,7 @@ async def update_game_stats(self):
                     mapId=self.map_id,
                     gameName=self.name,
                     validity=self.validity.value,
```

Comment:
I think the point of the issue is that the collection of the start time and the database insert should be separated. As in it should be possible to move the database insert to an earlier or later time and the start time should still be recorded correctly.